Sources/SwiftInspectorVisitors/Tests/PropertyVisitorSpec.swift

```diff
@@ -258,6 +258,91 @@ final class PropertyVisitorSpec: QuickSpec {
         }
       }
 
+      context("when the property is an undefined constant property") {
+        let content = """
+        let foo: Foo
+        """
+
+        it("has expected paradigm") {
+          try sut.walkContent(content)
+          expect(sut.properties) == [
```

Comment:
Maybe writing this as `sut.properties.first?.paradigm` and checking on the specific property might be more robust and resilient to refactors?

Reply:
https://github.com/fdiaz/SwiftInspector/pull/112/commits/095be15822fcf16b1aa1b86d7817f6ea353480aa